app/views/authors/index.html.haml

```diff
@@ -0,0 +1,11 @@
+.authors
+  - @authors.each do |author|
+    .author
+      .name
+        = author.first_name
+        = author.last_name
+      .biography= author.biography
+      %ul.author-books
+        - author.last_5_books(author).each do |book|
+          %li.book= image_tag book.cover_url
```

Comment:
@Alexneisc you will have N+1 query problems here. Because you are loading `Authors` in one query, and then for each author you will load 5 books. If you will show 100 authors, then you will make 101 query.
It is ok for now, but we will talk about it on meeting.